### ISSUE TITLE
Releasing the code for training Mask R-CNN or Cascade Mask R-CNN

### ISSUE THREAD

**Issue by atti0127** [EdwardChasel/BinaryAttention#4]
Thanks for your nice work !

By the way, may I ask you to release the code for training Mask R-CNN or Cascade Mask R-CNN using deit as backbone? (Table 2)

**Comment by EdwardChasel** [comment]
We conducted the detection tasks using [Detectron2](https://github.com/facebookresearch/detectron2). In these experiments, we fine-tuned the models with the pre-trained backbone weights and did not adopt self-distillation strategies. As for the other experimental setups, we mainly referenced the [Vim](https://github.com/hustvl/Vim/tree/main/det) implementation.
